lms/djangoapps/verify_student/views.py

```diff
@@ -1006,6 +1007,20 @@ def results_callback(request):
         log.debug("Approving verification for %s", receipt_id)
         attempt.approve()
         status = "approved"
+        checkpoints = VerificationCheckpoint.objects.filter(photo_verification=attempt).all()
+        if checkpoints:
+            course_key = checkpoints[0].course_id
+            credit_requirement = CreditRequirement.get_course_requirement(
+                course_key, "reverification", checkpoints[0].checkpoint_name
```

Comment:
VerificationCheckpoint model is being updated by Zubair. checkpoint_name field will be checkpoint_location so you would have to update "checkpoints[0].checkpoint_name" line once that code is merged.